clang/test/OpenMP/distribute_parallel_for_simd_if_codegen.cpp

```diff
@@ -2567,16 +2567,16 @@ int main() {
 // CHECK3-NEXT:    [[I:%.*]] = alloca i32, align 4
 // CHECK3-NEXT:    [[DOTCAPTURE_EXPR__CASTED:%.*]] = alloca i64, align 8
 // CHECK3-NEXT:    [[DOTBOUND_ZERO_ADDR:%.*]] = alloca i32, align 4
-// CHECK3-NEXT:    [[DOTCAPTURE_EXPR__CASTED12:%.*]] = alloca i64, align 8
```

Comment:
Please regenerate the OpenMP tests in a separate pull request, to reduce the number of irrelevant changes.

Reply:
They failed due to the change that generalized handling of `bool` and `_BitInt` in `EmitToMemory`/`EmitFromMemory`. I will need to revert that change then.

Reply:
It looked like some of the changes weren't related... if they're all in fact related, it's fine, I guess.